FirebaseDynamicLinks/Sources/Public/FirebaseDynamicLinks/FIRDynamicLink.h

```diff
@@ -67,6 +67,12 @@ NS_SWIFT_NAME(DynamicLink)
  */
 @property(nonatomic, assign, readonly) FIRDLMatchType matchType;
 
+/**
+ * @property utmParametersDictionary
+ * @abstract UTM parameters associated with a firebase dynamic link.
```

Comment:
`Firebase`, and should `dynamic link` be capitalized?

Reply:
Done